### ISSUE TITLE
Integration with Daily Notes

### ISSUE THREAD

**Issue by TheMathGuyd** [davish/obsidian-full-calendar#63]
Allow a clickable label for each day that links for daily notes,
additionally, add todays events to said daily note.

for instance clicking on the  ``15`` in the month view of February leads to ``[[2022-02-15]]``. This file can include a section for events pulled from FullCalendar

```
...

## Calendar
- 8:00-10:00 [[Commute]]
- 12:00-13:00 [[Call with Reggie]]
- 18:00-19:00 [[Drive home]]

...
```

This for additional notes to be added **between** these events without the need for cluttering up the event files themselves.

**Comment by davish** [comment]
This is pretty similar to what is being tracked in #28. I understand that this request would still have events with their own notes, but considering that the bullet points would have to move around to different daily notes when events get edited, it's probably going to be harder than #28 to implement.

#28 is one of the next features on my to-do list, so hopefully I'll get to it soon!

**Comment by 0xFlo** [comment]
Please let us know when this will be available. I'm only using the calendar to create daily notes, for everything else I have a calendar app/ google calendar standalone.

**Comment by davish** [comment]
@0xFlo this is now available in `0.9.0`. Cheers!
